embedded-google-storage/src/main/java/com/playtika/test/storage/EmbeddedStorageBootstrapConfiguration.java

```diff
@@ -0,0 +1,117 @@
+package com.playtika.test.storage;
+
+import com.playtika.test.common.spring.DockerPresenceBootstrapConfiguration;
+import com.playtika.test.common.utils.ContainerUtils;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.AutoConfigureAfter;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnExpression;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.env.ConfigurableEnvironment;
+import org.springframework.core.env.MapPropertySource;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.shaded.okhttp3.MediaType;
+import org.testcontainers.shaded.okhttp3.OkHttpClient;
+import org.testcontainers.shaded.okhttp3.Request;
+import org.testcontainers.shaded.okhttp3.RequestBody;
+import org.testcontainers.shaded.okhttp3.Response;
+
+import java.io.IOException;
+import java.util.LinkedHashMap;
+
+import static com.playtika.test.common.utils.ContainerUtils.configureCommonsAndStart;
+import static java.lang.String.format;
+
+@Slf4j
+@Configuration
+@ConditionalOnExpression("${embedded.containers.enabled:true}")
```

Comment:
Please add 
`@ConditionalOnClass(com.google.cloud.storage.Storage.class)`
to this autoconfiguration, so that is does not startup and fail if user does not have google-cloud-storage on classpath.